app/Domain/Consignment/Services/generateInvoiceService.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Domain\Consignment\Services;
+
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+use LaravelDaily\Invoices\Classes\InvoiceItem;
+use LaravelDaily\Invoices\Classes\Party;
+use LaravelDaily\Invoices\Invoice;
+
+class generateInvoiceService
+{
+    public function generateInvoice($consignment, $consignment_products): Invoice
```

Comment:
Try to refactor this class a little bit.
Extract parts of the code into methods